package.json

```diff
@@ -22,9 +22,9 @@
     "start": "node build/index.js",
     "prepare": "is-ci || husky",
     "format": "npx prettier --write .",
-    "docs:build:dev": "npx typedoc --watch",
     "docs:build": "npx typedoc",
-    "docs:serve": "PORT=4000 npx serve docs",
+    "docs:build:watch": "npm run docs:build -- --watch",
```

Comment:
I reverted these changes from `main` to introduce them at https://github.com/onebeyond/agnostic-cms-harmonizer/pull/22
Maybe you need to pull this branch from `main`?

Reply:
Not sure what you mean @Bounteous17. Since you flagged the CPU usage issue fixed in your branch, I merged these scripts into this one.